### DISCUSSION TITLE
Closed vs. Open Software and Civil Society

### DISCUSSION THREAD

**Discussion by epham36** [albertkun/21S-ASIAAM-191A#103] [Past Thinking Caps]
Open source tools can be especially helpful for digital mapping since the accessibility of the software allows for the exchange of ideas and solutions to problems globally -- especially for a topic with as global of a scope as mapping. The increased accessibility of software to users, forums, and answer-seekers makes it possible for people of all skill levels (be it professional geographers and cartographers or volunteers and laypeople) to make use of this data infrastructure. The transparency of the tools used grant people greater knowledge about the reliability and validity of their utilities -- and whether to, as well as how to, improve the tool itself -- a luxury not provided to closed source data.

Moreover, the accessibility of open source software makes it an attractive tool for community-based organizations which may not have the budget for proprietary software that organizations in the private and public sector have. Open source software poses an alternative to proprietary, closed-source software that is frequently updated -- a quick turnover that is incentivized by commercial interests. They pose alternatives to corporate software that are widely used by the mainstream, which often are falsely framed as the sole choices for software uses. 

These modes of software that promote collaborative forms of software production challenge corporatist, capitalist, and individualistic notions of software production and knowledge distribution. Rather than siloing software and knowledge prouction into individual corporations, open and free software are a political statement that firmly asserts the choice of who has access to these tools. Pitted against closed source software, open and free software are tools of moral agency.

Certainly, closed source software allow for additional efficiency and reliability, as these softwares are continuously upkept and made up to date in order to stay on top of commercial interests. The capitalistic drive for success, excellency, and exclusivity incentivize constant updates and maintenance checks to ensure speed of the software, (at times) planned obsolescence of previous software, and outcompeting of competitors. Whereas, open-sourced software, in its more collaborative spirit, prioritizes efficiency and state-of-the-art cutting edge technology to be secondary to accessibility and collaboration between users -- which inherently moves at a slower pace in order to provide access to general members of the public and uplift community voices over esoteric software developers. 

Moreover, the use of open source software may not always be compatible with the more mainstream, closed-source software -- and users of these softwares may not find their software to be fully compatible with closed-source competing software that their colleagues may be using.

**Comment by ghost** [comment]
I really enjoyed your explanation of open source tools and think you perfectly described why they are so attractive for socially driven projects, collaboration, and more! I think it is important to recognize, as you pointed out, that open sources have a lot of benefits but can fall short of the technical innovation and reliability of closed source platforms. When looking between the two and choosing which to use (if privileged enough to have the option of closed source) I think that open source is almost always the ideal choice for me because of what it stands for. I am more than willing to sacrifice a little technical innovation and software speed in order to create maps and more on a platform that is free and accessible to everyone (even if not everyone uses it in positive ways).

**Comment by avaitt** [comment]
I loved your explanation! I think your description of the pros and cons of both open and closed sources was very thorough. I never realized that open-source software can sometimes stifle progress compared to closed-source solutions. The point you made about compatibilities across software was also interesting. We need to definitely find a middle-way that will promote the efficiencies and dependabilities fo closed-source with the collaboration of open-source.

**Comment by nnhimichelle** [comment]
Thank you for this explanation, I enjoyed reading it! I think your description of closed-source software and tying it into capitalism has shown significantly exclusivity of a variety of communities when putting up services at a cost. Not only that, it also shows how deeply ingrained it is into our collective consciousness that the little things, such as these services, just simply reinforce its place in society. I also like how you pointed the different benefits that open-source software provides and how community organizations can utilize it.
